version/version.go

```diff
@@ -31,7 +32,20 @@ type ServerVersioner interface {
 	ServerVersion() (*version.Info, error)
 }
 
-var minimumVersion = "v1.11.0"
+const (
+	// MinimumKubernetesKey is the environment variable that can be used to override
+	// the minimum Kubernetes version required by Knative.
```

Comment:
nit: for consistency with EnvVar

```suggestion
	// the Kubernetes minimum version required by Knative.
```

Reply:
done